---
float64[16] o_T_ee  # row major
